tests/test_lit_server.py

```diff
@@ -17,14 +16,14 @@ def test_new_pipe(lit_server):
     for _ in range(pool_size):
         lit_server.new_pipe()
 
-    assert len(lit_server.pipe_pool) == 0
-    assert len(lit_server.new_pipe()) == 2
+    assert len(lit_server.pipe_pool) == 0, "pipe_pool was completely used and need to be empty"
```

Comment:
can you describe it in non technical terms as well?
- what in the code caused this issue?
- etc...

Reply:
@williamFalcon updated to

```py
assert len(lit_server.pipe_pool) == 0, "lit_server.new_pipe() was used till max_pool_size which exhausts the pool"
assert len(lit_server.new_pipe()) == 2, "lit_server.new_pipe() always must return a tuple of read and write pipes"
```